gc/base/GCExtensionsBase.cpp

```diff
@@ -182,10 +182,12 @@ MM_GCExtensionsBase::initialize(MM_EnvironmentBase* env)
 	if (J9HookInitializeInterface(getPrivateHookInterface(), OMRPORTLIB, sizeof(privateHookInterface))) {
 		goto failed;
 	}
+	INITIALIZE_HOOKDUMP(privateHookInterface);
```

Comment:
yes, getPrivateHookInterface() == privateHookInterface, but J9HookInitializeInterface() cast it to common pointer ( struct J9HookInterface ** ), I could not figure out which HookInteface inside method J9HookInitializeInterface()